frontend/test/metabase/components/TokenField.unit.spec.js

```diff
@@ -142,7 +157,7 @@ describe("TokenField", () => {
       />,
     );
     userEvent.type(input(), "yep");
-    expect(input().value).toEqual("");
+    expect(input().value).toEqual("yep");
```

Comment:
I change this because I think this is the _correct_ expectation.

Reply:
How did it even pass before this change…

Reply:
Because before this PR, it threw an exception and therefore it led to the incorrect expectation.